data_resource/config/configuration.py

```diff
@@ -148,25 +152,63 @@ def __init__(self):
 
 
 class ProductionConfig(Config):
-     """Production deployment configuration class."""
-
-     def __init__(self):
-         super().__init__()
-
-     SQLALCHEMY_TRACK_MODIFICATIONS = False
-     PROPAGATE_EXCEPTIONS = True
-     POSTGRES_USER = os.getenv("POSTGRES_USER")
-     POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
-     POSTGRES_DATABASE = os.getenv("POSTGRES_DATABASE")
-     POSTGRES_HOSTNAME = os.getenv("POSTGRES_HOSTNAME", "localhost")
-     POSTGRES_PORT = os.getenv("POSTGRES_PORT", 5432)
-     SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://{}:{}@{}:{}/{}".format(
-        POSTGRES_USER,
-        POSTGRES_PASSWORD,
-        POSTGRES_HOSTNAME,
-        POSTGRES_PORT,
-        POSTGRES_DATABASE,
-     )
+    """Production deployment configuration class."""
```

Comment:
Are there any regression tests that cover this behavior?

I'd like to have some kind of assurance that when we deploy to production that any changes to the code didn't break this.

Reply:
Nope, there is no regression test since it pings aws sdk.

Reply:
Added this as a back log item -- https://app.clubhouse.io/brighthive/story/1261/test-for-drg-to-access-credentials-via-aws